1 Holiday Rental Property Management Tool/submissions/enricop89/README.md

```diff
@@ -0,0 +1,21 @@
+# Holiday Rental Property Management Tool
+
+For this project I imagined to host the client side application in an S3 bucket backed by a Cloudfront distribution to enforce HTTPS, global distribution and reduced latency. The client side application will have two main users: guests and hosts. Hosts can login using AWS Cognito, Guests can only see the property and book a room.
```

Comment:
Good question - I imagine that when a guest books a property, it will provide a few details such as email, firstname,lastname. In the confirmation email, I can send a booking confirmation code linked to his email, something similar to what airlines do.